### PR TITLE
20.10.0 changelog

### PR DESCRIPTION
**Pull request by timbeiko**
Changelog for the 20.10.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -53,11 +41,12 @@ Hyperledger Besu is moving its versioning scheme to [CalVer](https://calver.org/
 - [Privacy users with private transactions created using v1.3.4 or earlier](KNOWN_ISSUES.md#privacy-users-with-private-transactions-created-using-v134-or-earlier)
 - [Changes not saved to database correctly causing inconsistent private states](KNOWN_ISSUES.md#Changes-not-saved-to-database-correctly-causing-inconsistent-private-states)
```
Comment (MadelineMurray) [inline CHANGELOG.md:42]: @lucassaldanha and @pinges - this was fixed wasn't it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,8 @@
 # Changelog
 
-## Deprecated and Scheduled for removal in _Next_ Release
+## Release format
 
-### --privacy-precompiled-address
-Deprecated in 1.5.1
-- CLI option `--privacy-precompiled-address` option removed. This address is now derived, based
-on `--privacy-onchain-groups-enabled`. [\#1222](https://github.com/hyperledger/besu/pull/1222)
+Hyperledger Besu is moving its versioning scheme to [CalVer](https://calver.org/) starting with the 20.10.0 (f.k.a. 1.6.0) release. More information about the specific version of CalVer Besu is using can be found on the [wiki](https://wiki.hyperledger.org/display/BESU/Proposal+-+CalVer+for+Besu+Releases).
```
[the remark this answers is not in the document]
Comment (timbeiko) [reply CHANGELOG.md:5]: Yes, good call.

[the remark this answers is not in the document]
Comment (timbeiko) [reply CHANGELOG.md:5]: No, but I've updated the Wiki post. We should use the latest version: https://wiki.hyperledger.org/display/BESU/Using+CalVer+for+Besu+Releases

File: CHANGELOG.md (hunk 1 above)
Comment (pinges) [reply CHANGELOG.md:42]: Yes, that seems to be fixed by our changes.

**Review by arash009 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,8 @@
 # Changelog
 
-## Deprecated and Scheduled for removal in _Next_ Release
+## Release format
 
-### --privacy-precompiled-address
-Deprecated in 1.5.1
-- CLI option `--privacy-precompiled-address` option removed. This address is now derived, based
-on `--privacy-onchain-groups-enabled`. [\#1222](https://github.com/hyperledger/besu/pull/1222)
+Hyperledger Besu is moving its versioning scheme to [CalVer](https://calver.org/) starting with the 20.10.0 (f.k.a. 1.6.0) release. More information about the specific version of CalVer Besu is using can be found on the [wiki](https://wiki.hyperledger.org/display/BESU/Using+CalVer+for+Besu+Releases).
```
Comment (MadelineMurray) [inline CHANGELOG.md:5]: I had to think for a second to figure out f.k.a -> writing in full might be better

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,20 +16,11 @@ In Besu version 20.10, properly formatted requests that have valid parameters (c
 
 Users of Web3J should note that many calls will now return a result with the error field containing the message whereas before a call would throw an exception with the error message as the exception message.
```
Comment (MadelineMurray) [inline CHANGELOG.md:17]: Not sure who the best person is to talk to about this but will web3j users need a list of specific calls or a link to the GI that changed this to be able to assess any impact to them?

**Review by MadelineMurray — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (MadelineMurray) [reply CHANGELOG.md:17]: Thanks - that adds the context.

**Pull request merged**